staff/lab/anotha

```diff
@@ -0,0 +1,21 @@
+if [[ -z $1 ]]; then
```

Comment:
I'd have `#!/bin/bash` at the top here, just to prevent against those sneaky errors, and have `set -euo pipefail` under it to make bash just a bit more sane.